add unit test

AlirezaEm opened the pull request:
Unit test has been implemented for the get_current_user function and also for 3 of the endpoints: 
# POST /v1/orders : Create an empty shopping cart for the user
# DELETE /v1/orders/uuid
Endpoint to get orders based on user and/or state : 
    #GET /v1/orders?user=uuid&state=SHIPPED|PAID|etc  Get all shipped orders for a user by state
    # GET /v1/orders?user=uuid  Get all orders of a user
    #GET /v1/orders?state=PAID  Get all “Paid” orders for ALL users

Commit:
add unit test

WillBkWhiteFoxCloud reviewed (commented):
1. Please add tests for checkout_shopping_cart and update_shopping_cart as well.
2. If clone this repo in another computer and install requirements, you will see that the tests will fail because some development dependencies are missing. For example pytest and others. Please create requirements-dev.txt for these dependencies.
3. Please update your Readme file to guide user how to test your app.

Commit:
add test for update and checkout routes, and also development requirements text file

Commit:
add The httpx command line client could not run because the required dependencies were not installed.
Make sure you've installed everything with: pip install 'httpx[cli]' package as a dev requirement and edit README

Commit:
add AWS_DEFAULT_REGION to monkeypatch and add moto library to the dev requirements

AlirezaEm committed:
Update README.md
